web/pages/[org]/[repo]/pulls/[num]/owners.tsx

```diff
@@ -1,50 +1,55 @@
-import { Card } from "antd";
+import { Tag, Descriptions } from "antd";
 import React from "react";
 import { get } from "@/utils/request";
-import {
-  BASE_URL,
-  COMMITTER,
-  GITHUB_BASE_URL,
-  REVIEWER,
-} from "@/types/constant";
-import { IOwnerTypeData } from "@/types/owners";
+import { BASE_URL, GITHUB_BASE_URL } from "@/types/constant";
+import { OwnersData } from "@/types/owners";
 
 import style from "./owners.module.scss";
 
-export default function Owners({ owners }) {
-  const detailData: Partial<IOwnerTypeData> = owners.data;
+export default function Owners({ org, repo, num, owners }) {
+  const ownersData: Partial<OwnersData> = owners.data;
 
-  const renderMembers = (members: Array<string>, title: string) => {
+  const renderMembers = (members: Array<string>) => {
     return (
-      <>
-        <p className={style.member}>{title}</p>
-        <div className={style.wrapper}>
-          {members.map((member) => (
-            <Card
-              className={style.container}
-              cover={<img alt="pic" src={`${GITHUB_BASE_URL + member}.png`} />}
-            >
-              <a href={GITHUB_BASE_URL + member}>{member}</a>
-            </Card>
-          ))}
-        </div>
-      </>
+      <p className={style.members}>
+        {members.map((member, index) => (
+          <a key={member} href={GITHUB_BASE_URL + member}>
+            {" "}
```

Comment:
It doesn't look very elegant, but it can also be accepted.

Reply:
Refactored.